controllers/policyset/policyset_controller.go

```diff
@@ -330,21 +349,26 @@ func (r *PolicySetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 // Helper function to filter out compliance statuses that are not in scope
 func complianceInRelevantClusters(status []*policyv1.CompliancePerClusterStatus, relevantClusters []string) string {
```

Comment:
Optional: Ideally, this would return `ComplianceState` instead of a string. In my opinion, we should keep it typed until we really need to convert it to a string.